apps/docs/src/content/docs/reference/default.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: Default task
+description: the task function
+---
+
+import { LinkCard, Tabs, TabItem, Aside } from '@astrojs/starlight/components';
+
+This is the way to create a default task.
+
+There's no concurrency management in this task and every `perform` will be executed immediately.
+
+## Usage
+
+To specify a default task, you can either use the dot notation or the options notation.
+
+<Tabs syncKey="notation">
+
+<TabItem label="Dot notation">
+
+```svelte
+<script lang="ts">
+	import { task } from '@sheepdog/svelte';
+
+	const myTask = task(async () => {
+		// your code
+	});
+</script>
+```
+
+</TabItem>
+
+<TabItem label="Options notation">
+
+```svelte
+<script lang="ts">
+	import { task } from '@sheepdog/svelte';
+
+	const myTask = task(
+		async () => {
+			// your code
+		},
+		{ kind: 'default' },
+	);
+</script>
+```
+
+</TabItem>
+
+</Tabs>
+
+### Max concurrency
+
+Since this kind of task does not do any concurrency management there's no way to specify a `max`
+concurrency value.
```

Comment:
I think this section should be removed or moved at the very end of the page as a simple note because it's somehow confusing to dedicate a section to something you cannot do.

Reply:
I think i agree...will move it to the end as a warning